consent.yaml

```diff
@@ -1444,3 +1713,13 @@ components:
           $ref: '#/components/schemas/ConsentEventType'
         example: 'WEBHOOK_SENDING_FAILED,WEBHOOK_SENDING_FAILED_CLIENT_ERROR'
       description: Consent event types to be returned
+    QueryParamPageSize:
+      name: pageSize
+      in: query
+      required: false
+      schema:
+        type: integer
+        minimum: 1
+        default: 1000
+        example: 1000
+      description: Page size to request. Default is 1000
```

Comment:
this looks like a dupe of https://github.com/Adatree/api-specs/pull/47/files#diff-fa21ef9cc73e0385ed1e3d52420c3d0871b0b4880678a5e40eebec4d18b11543R1660-R1668

Reply:
different default parameter... :(